app/lib/common/widget/customAvatar.dart

```diff
@@ -0,0 +1,69 @@
+import 'dart:typed_data';
+import 'package:flutter/material.dart';
+import 'package:effektio/common/store/Colors.dart';
+
+class CustomAvatar extends StatelessWidget {
+  final Future<List<int>> avatar;
+  final Future<String> displayName;
+  final double radius;
+  const CustomAvatar({
+    Key? key,
+    required this.radius,
+    required this.avatar,
+    required this.displayName,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return FutureBuilder<List<int>>(
+      future: avatar,
+      builder: (
+        BuildContext context,
+        AsyncSnapshot<List<int>> snapshot,
+      ) {
+        if (snapshot.hasData) {
+          return Container(
+            margin: const EdgeInsets.all(10),
+            child: CircleAvatar(
+              backgroundImage: MemoryImage(
+                Uint8List.fromList(
+                  snapshot.requireData,
+                ),
+              ),
+              radius: radius,
+            ),
+          );
+        } else {
+         return FutureBuilder<String>(
+            future: displayName,
+            builder: (
+              BuildContext context,
+              AsyncSnapshot<String> snapshot,
+            ) {
+              if (snapshot.hasData) {
+                return Container(
+                  margin: const EdgeInsets.all(10),
+                  child: CircleAvatar(
+                    backgroundColor: Colors.brown.shade800,
```

Comment:
we should do something more clever than just have all avatars be brown. e.g. what [colorized avatar](https://flutterkit.dev/flutter-plugin-that-allows-users-to-create-text-avatar-easily/) does...

Reply:
Yes, shall I close this request and start a new branch related to the main branch and make these changes there?

Reply:
let's do that here first, please. there's still other pending things anyways before we can merge this.